### DISCUSSION TITLE
Any plans to also support opnsense?

### DISCUSSION THREAD

**Discussion by ssbarnea** [pfsensible/core#45] [Q&A]
As I switched to opnsense few years back, I wonder if this project would aim to cover both pfsense and opnsense as the number of differences are not so big.

**Comment by opoplawski** [comment]
I'm curious how you determined that the differences are not so big.  Since pfsensible directly edits the config.xml file and calls php functions, it's pretty sensitive to any differences.   Personally, I would hate to see the pfsensible code littered with lots of conditionals handling the differences.  Finally, since I don't use opnsense, I don't have any motivation to support it.

That said, anyone is free to fork this and start playing around to see just how hard it would be to support both.

**Reply by ssbarnea** [reply]
Thanks for the quick response. Yep, the xml "blob" is a big historical mistake that hinders implementation of good configuration management, which should really use rest api calls to change different settings... api calls both never had.
